docs/source/vagrant.rst

```diff
@@ -74,6 +74,8 @@ Install
 
       cd /vagrant/cozmo_sdk_examples_***
 
+    .. important:: You must either relaunch with "vagrant up" or save your virtual machine's state when shutting down.  Otherwise the /vagrant/ folder on the virtual machine will be empty on subsequent runs.
```

Comment:
Instead of this information being duplicated in both Windows and Mac steps 5/6, might be better to just have this as a note e.g. after step 9